frontend/src/employee-frontend/components/application-page/ApplicationReadView.tsx

```diff
@@ -143,12 +144,14 @@ export default React.memo(function ApplicationReadView({
       applicationGuardian.residenceCode === otherGuardian.residenceCode) ||
     false
 
-  const attachmentReceivedAt = (attachment: ApplicationAttachment): Date => {
+  const attachmentReceivedAt = (
+    attachment: ApplicationAttachment
+  ): LocalDate => {
     const { sentDate } = application.application
-    if (!sentDate) return attachment.receivedAt
-    else if (sentDate.toSystemTzDate() > attachment.receivedAt)
-      return sentDate.toSystemTzDate()
-    else return attachment.receivedAt
+    if (!sentDate) return attachment.receivedAt.toLocalDate()
+    else if (sentDate.isEqualOrAfter(attachment.receivedAt.toLocalDate()))
```

Comment:
The original version reads as sentDate.isAfter(receivedAt), not isEqualOrAfter

Reply:
True, the change is unnecessary and confusing...and it doesn't actually matter since we just want max of the two values. I've replaced this if-else with a maxOf function that is IMHO clearer